### DISCUSSION TITLE
Feature Request - Allow saved draft response to be visible to other Agents

### DISCUSSION THREAD

**Discussion by GregoryPost** [osTicket/osTicket#6582] [Ideas]
Sometime it's nice to write an initial response to a ticket and then allow other team members to weigh in before sending the reply.  osTicket allows for saving draft replies to a ticket, however - as far as I know - this draft is only visible to the agent that wrote/saved it.  Requesting a feature to allow other agents that have access to the ticket the ability to view, edit, and ultimately send a saved draft reply.

**Comment by JediKev** [comment]
@GregPost-ASU 

Can't you just post an Internal Note with the suggested message and have agents weigh-in?

Cheers.
